src/coreComponents/physicsSolvers/wavePropagation/AcousticWaveEquationSEM.cpp

```diff
@@ -343,10 +362,12 @@ void AcousticWaveEquationSEM::computeSeismoTrace( real64 const time_n,
           // Note: this "manual" output to file is temporary
           //       It should be removed as soon as we can use TimeHistory to output data not registered on the mesh
           // TODO: remove saveSeismo and replace with TimeHistory
+          std::ofstream f( GEOSX_FMT( "seismoTraceReceiver{:03}.txt", ircv ), std::ios::app );
           for( localIndex iSample = 0; iSample < m_nsamplesSeismoTrace; ++iSample )
           {
-            this->saveSeismo( iSample, varAtReceivers[iSample][ircv], GEOSX_FMT( "seismoTraceReceiver{:03}.txt", ircv ) );
+            f << iSample << " " << varAtReceivers[iSample][ircv] << std::endl;
           }
+          f.close();
```

Comment:
Yes, I just moved the function body into here to avoid opening and closing the output file at every time sample.
I know this way to output results is temporary, but still.